includes/blocks/course-theme/class-ui.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Learning Mode Ui block.
+ *
+ * @package sensei
+ * @since
+ */
+
+namespace Sensei\Blocks\Course_Theme;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+use \Sensei_Blocks;
+
+/**
+ * User interface block for Learning mode layout elements.
+ */
+class Ui {
+	/**
+	 * Block JSON file.
+	 */
+	const BLOCK_JSON_FILE = '/ui/ui.block.json';
+
+	/**
+	 * Course_Title constructor.
+	 */
+	public function __construct() {
+		$block_json_path = Sensei()->assets->src_path( 'course-theme/blocks' ) . self::BLOCK_JSON_FILE;
+		Sensei_Blocks::register_sensei_block(
+			'sensei-lms/ui',
+			[
+				'render_callback' => [ $this, 'render' ],
```

Comment:
Should we have a `render` method in this class?

Reply:
Forgot to remove this, for now it doesn't need a render callback.